source/administration/production-notes.txt

```diff
@@ -740,20 +778,51 @@ accommodate Windows' commitment rules during peak database use.
 MongoDB on Virtual Environments
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This section describes considerations when running MongoDB in some of the
-more common virtual environments.
+This section describes considerations when running MongoDB in some of 
+the more common virtual environments.
 
 For all platforms, consider :ref:`virtualized-disks-scheduling`.
 
-EC2
-```
+:abbr:`AWS (Amazon Web Services)` :abbr:`EC2 (Elastic Compute Cloud)`
+`````````````````````````````````````````````````````````````````````
+
+There are two performance configurations to consider:
+
+- Reproducible performance for performance testing or benchmarking, and
+- Raw maximum performance
+
+To tune performance on :abbr:`EC2 (Elastic Compute Cloud)` for either 
+configuration, you should:
+
+- Enable :abbr:`AWS (Amazon Web Services)`
+  `Enhanced Networking <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking>`_ 
+  for your instance. Not all instance types support Enhanced Networking. 
+
+  To learn more about Enhanced Networking, see to the 
+  `AWS documentation <http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/enhanced-networking.html#enabling_enhanced_networking>`_.
+
+- Use provisioned :abbr:`IOPS (Input/Output Operations Per Second)` for 
+  the storage, with separate devices for journal and data. Do not use
+  the ephemeral storage available on some instance types as their
```

Comment:
"Some instance types" is vague.

While we didn't spend time going in that direction, for example the i-family of instance types has very good SSDs which should far outperform anything EBS can do. (It's also a very expensive instance type.)

Reply:
I might move the part about ephemeral storage, and the next part about dvfs below the "If you are concerned about reproducible performance" part. To make a weak statement, I know they apply there, I don't know they apply to the performance case (they might).

Reply:
BTW, I like the split between the two scenarios. That looks good.